bin/genconfig2

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib_dir)
 
 require 'beaker-hostgenerator'
 
-STDERR.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
+$stderr.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
```

Comment:
From what I can see, there are no direct references to STDOUT (or $stdout) in the lib, so I think we should be good here